source/cipher.c

```diff
@@ -976,6 +979,9 @@ int aws_cryptosdk_rsa_decrypt(
     const struct aws_byte_cursor cipher,
     const struct aws_string *rsa_private_key_pem,
     enum aws_cryptosdk_rsa_padding_mode rsa_padding_mode) {
+    AWS_PRECONDITION(aws_byte_buf_is_valid(plain));
+    AWS_PRECONDITION(aws_byte_cursor_is_valid(&cipher));
+    AWS_PRECONDITION(rsa_private_key_pem != NULL);
```

Comment:
Why can't we assume here `aws_string_is_valid()`?

Reply:
Updated!